api/include/opentelemetry/trace/experimental_semantic_conventions.h

```diff
@@ -183,7 +183,7 @@ static const struct
 
 inline const char *attr(uint32_t attr)
 {
-  for (int i = 0; i < OTEL_CPP_TRACE_ATTRIBUTES_MAX; i++)
+  for (long unsigned int i = 0; i < OTEL_CPP_TRACE_ATTRIBUTES_MAX; i++)
```

Comment:
What is the type `long unsigned int`? Probably `unsigned int`?

Reply:
it was the type compiler was showing. I changed to `size_t`.